src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DSDropDownBase.cs

```diff
@@ -38,6 +38,11 @@ public void CustomizeView(DSDropDownBase model, NodeView nodeView)
             System.Windows.Controls.Grid.SetRow(combo, 0);
 
             combo.DropDownOpened += combo_DropDownOpened;
+            combo.SelectionChanged += delegate
```

Comment:
please use a named handler for this and unsubscribe it in the dispose call. It's likely this usage is safe, but it's usually simpler and more maintainable in the long run to just unsubscribe ALL event handlers at a known life cycle hook instead of proving that they are safe with a profiler.

One day we'll have memory unit tests, until then...

Reply:
Ah, makes sense. I was adding back code removed in the other PR, but happy to improve it.

Reply:
While we're cleaning up - I went through and did all the obvious clean-up in that file. Let me know if I went too far though!